docs/content/deployment/guides/kubernetes/customizing-your-deployment.mdx

```diff
@@ -116,11 +116,11 @@ def my_job():
 
 Other run launchers will ignore the `dagster-k8s/config` tag.
 
-If your Dagster job is configured with the <PyObject module="dagster_k8s" object="k8s_job_executor" /> that runs each step in its own pod, configuration that you set on a job using the `dagster-k8s/config` tag will _not_ be propagated to any of those step pods.
+The default <a href="/deployment/executors">executor</a> <PyObject module="dagster" object="multi_or_in_process_executor" /> will run each job in its own pod executing each step in an individual process. If your Dagster job produces assets which have a short compute time (compared to the step overhead time), consider avoiding the step process creation cost by using the <PyObject module="dagster" object="in_process_executor" /> executor, which runs each step serially in a single process. This can be especially useful where parallelism is obtained through a <PyObject object="PartitionsDefinition" /> and is unnecessary within the job. For this use-case, the <PyObject module="dagster" object="in_process_executor" /> is more efficient than running each step in its own process and the <PyObject module="dagster_k8s" object="k8s_job_executor" /> is contraindicated as the delay of scheduling and starting up a new Dagster pod to execute every step would significantly slow down overall execution time.
```

Comment:
```suggestion
The default [executor](/deployment/executors) - <PyObject module="dagster" object="multi_or_in_process_executor" /> - will run each job in its own pod, executing each step in an individual process. If your Dagster job produces assets which have a short compute time (compared to the step overhead time), consider avoiding the step process creation cost by using the <PyObject module="dagster" object="in_process_executor" /> executor, which runs each step serially in a single process. This can be especially useful where parallelism is obtained through a <PyObject object="PartitionsDefinition" /> and is unnecessary within the job. 

For this use-case, the <PyObject module="dagster" object="in_process_executor" /> is more efficient than running each step in its own process. The <PyObject module="dagster_k8s" object="k8s_job_executor" /> is contraindicated, as the delay of scheduling and starting up a new Dagster pod to execute every step would significantly slow down overall execution time.
```

Using suggestions to make things easier to communicate.

- No need for HTML links here - Markdown is fine
- Break second half of this into its own paragraph. I think combining some of this improved things, but it does need a visual (and reading!) break. This seems like the right place for it.
- Broke the last paragraph into multiple sentences. Good info, but it being on the long side can make it harder to read + understand.